modules/k6/build.gradle

```diff
@@ -0,0 +1,7 @@
+description = "Testcontainers :: k6"
```

Comment:
Grafana K6?

Reply:
Related to this as well: 
> Also, I think the module should be Grafana instead, later we can add more implementations for other Grafana products

I think this would add more confusion personally. k6 has primarily been known as an independent company/product from Grafana. k6 was an acquisition by Grafana Labs in 2021. WDYT?

Reply:
I did want to add to this point as well, that the [Go implementation](https://github.com/testcontainers/testcontainers-go/tree/main/modules/k6) and current [documentation](https://testcontainers.com/modules/k6/) reference "k6" and not "Grafana k6."